_episodes/09-pull-requests.md

```diff
@@ -133,17 +211,18 @@ $ git commit -m "Added file on france"
 {: .bash}
 
 ~~~
-[master 79a312a] Added file on france
+[addFrance 79a312a] Added file on france
  1 file changed, 2 insertions(+), 2 deletions(-)
 ~~~
 {: .output}
 
 Now we can push those changes to our forked version of the repo.
 Since we don't have permissions to the upstream repo or we want to suggest these changed be
```

Comment:
I made some changes here (will get pushed but copied below).  I edited your suggestion a little but I still feel like I missing the motivation a little here.  I want to get across that the pull request is useful for suggesting changes, esp if you don't have push permissions or want someone to review.  I might try working more on the language here.  Let me know if you have suggestions.

"Now we can push those changes to our forked version of the repo.
In some cases we may not have permission to push directly to the 
upstream repo or we might like our changes to be reviewed regardless 
of permissions, so we won't push directly to the upstream repository. 
Instead we will push the name of our new branch to the remote linked 
to our fork, `origin`."